ldes-server-infra-mongo/mongock-changesets/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/mongock/changeset5/MemberIngestChange.java

```diff
@@ -8,7 +8,8 @@
 import org.springframework.data.mongodb.core.MongoTemplate;
 import org.springframework.data.mongodb.core.query.Query;
 
-@ChangeUnit(id = "member-ingest-changeset-5", order = "5", author = "VSDS")
+// Not active yet, preparation for new architecture
+//@ChangeUnit(id = "member-ingest-changeset-5", order = "5", author = "VSDS")
```

Comment:
really not a big fan of "preparation for new architecture" since it's very vague. 
Do we have a timeline? a ticket? Let's put something more tangible than this :)